src/modules/gui/filesystem/home/pi/scripts/start_gui

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
-xset s off         # don't activate screensaver
-xset -dpms         # disable DPMS (Energy Star) features.
-xset s noblank     # don't blank the video device
+
+# Stop the screen from turning off when idle.
+DISABLE_POWER_MANAGEMENT=yes
```

Comment:
I've added these here rather than in config, as they need to be changed at runtime, maybe there is a better place to define these?